index.html

```diff
@@ -0,0 +1,104 @@
+<!DOCTYPE html>
+<html lang="es">
+
+<head>
+  <meta charset="utf-8" />
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1">
+  <link rel="stylesheet" type="text/css" media="screen" href="css/styles.css" />
+  <link href="https://fonts.googleapis.com/icon?family=Material+Icons" rel="stylesheet">
+  <link href="https://fonts.googleapis.com/css?family=Roboto:300,400,500,700" rel="stylesheet">
+  <link rel="stylesheet" href="https://use.fontawesome.com/releases/v5.6.3/css/all.css" integrity="sha384-UHRtZLI+pbxtHCWp1t77Bi1L4ZtiqrqD80Kn4Z8NTSRyMA2Fd33n5dQ8lWUE00s/"
+    crossorigin="anonymous">
+  <script src="main.js"></script>
+
+  <title>Intuit Accounts - Sign In</title>
+
+</head>
+
+<body>
+  <header class="header">
+    <div class="sign-up-container">
+      <div class="sign-textbox">
+        <div class="existing-account font-style letter-size">Already have an Intuit account?</div><br>
+        <div class="forgot-password font-style letter-size">I forgot my User ID or Password</div>
+      </div>
+      <span><button class="sign-in-button font-style">
+          <i class="material-icons">lock</i>
+          Sign In</button></span>
+    </div>
+  </header>
+
+  <header id="sign-in-form">
+    <img id="logo" class="center" src="img/intuit-logo.png" alt="intuit-logo" />
+  </header>
+  <div class="sign-up-form center">
+    <span id="slogan" class="font-style">
+      One Account.
+      <br>
+      Everything Intuit.
+    </span>
+    <div style="padding: 20px 0px 20px 0px">
+      <div id="create-text" class="font-style">
+        Create an account for all of Intuit's products.
+      </div>
+      <div class="learn-more font-style letter-size">
+        <a href="">Learn more</a>
+      </div>
+    </div>
+
+    <form class="main-form font-style">
+      <label for="email">Email address</label>
+      <div class="input-box"><input class="h-center" type="text" name="email" id="email"></div>
+
+      <label for="phone">Phone (recommended)</label>
+      <div class="input-box"><input class="h-center" type="text" name="phone" id="phone">
+        <p class="small-lbl">Standard call, messaging or data rates may apply.</p>
+      </div>
+
+      <label for="password">Password</label>
+      <div class="input-box"><input class="h-center" type="text" name="password" id="password"></div>
+
+      <br>
+
+      <button class="create-account-btn font-style" name="submit">
+        <i class="material-icons" style="text-decoration: none; color: white; font-size: 18px; line-height: 16px">lock</i>&nbsp;&nbsp;
```

Comment:
Same as previous comment. Avoid inline styles in your markup.
I know styling `i` elements can be tricky! One approach would be to wrap this inside a `span`, then styling that instead. e.g.
`<span class="icon"><i class="material-icons">lock</i></span>`.

Then, add the style to `.icon` class.